docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md

```diff
@@ -19,33 +19,41 @@ Each project stores blocks in different places and uses slightly different block
 
 For configuration of remote write to Grafana Mimir, refer to [Configuring Prometheus remote write]({{< relref "../operators-guide/secure/authentication-and-authorization.md#configuring-prometheus-remote-write" >}}).
 
-## Uploading historic blocks to the Grafana Mimir storage bucket
+## Uploading historic TSDB blocks to Grafana Mimir
+
+Grafana Mimir supports uploading of historic TSDB blocks, also from other systems such as Prometheus and Thanos.
```

Comment:
Let's only mention Prometheus, since we cannot upload Thanos blocks via `mimirtool backfill`.

Reply:
Done.